airgun/widgets.py

```diff
@@ -916,20 +916,25 @@ class CustomParameter(Table):
 
     add_new_value = Text("..//a[contains(text(),'+ Add Parameter')]")
 
-    def __init__(self, parent, locator=None, id=None, logger=None):
+    def __init__(self, parent, locator=None, id=None, logger=None, column_widgets=None):
```

Comment:
`Table` has a lot more kwargs than this, so the function signature here should be updated to take `**kwargs` and pull any references from there, instead of trying to maintain an exact signature with the parent class.

Reply:
I've attempted to address this but would appreciate feedback on my implementation.